.evergreen/build-mac.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash -ex
```

Comment:
Why are you adding this now since we can't actually do it?

Reply:
We can still use it to build macos wheels manually.